src/store/initialState.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+/**
+ * # Initial State
+ *
+ * This object contains initial state of the data store.
+ */
+module.exports = {
+    // List of entries (records) to visualize
+    entries: null,
+
+    // List D3 scales - one scale per axis
+    scales: null,
+
+    // List of attributes in the data set
+    // For numerical attributes also contains min/max values
+    // For categorical attributes also contains list of factors
+    attributes: null,
```

Comment:
Should `entries`, `scales`, and `attributes` be set to empty arrays instead of null? I'm wondering why these guys are null and `selectedEntries` and `snapshots` are arrays.